fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/service/LoanDownPaymentHandlerServiceImpl.java

```diff
@@ -172,8 +172,14 @@ public void handleRepaymentOrRecoveryOrWaiverTransaction(final Loan loan, final
             loan.updateLoanSummaryDerivedFields();
         }
 
+        BigDecimal disbursedThroughCapitalizedIncome = BigDecimal.ZERO;
+        if (loan.getLoanProductRelatedDetail().isEnableIncomeCapitalization()) {
+            disbursedThroughCapitalizedIncome = loan.getLoanTransactions().stream().filter(LoanTransaction::isCapitalizedIncome)
+                    .map(LoanTransaction::getPrincipalPortion).reduce(BigDecimal.ZERO, BigDecimal::add);
+        }
+
         if (loan.getLoanProduct().isMultiDisburseLoan()) {
-            final BigDecimal totalDisbursed = loan.getDisbursedAmount();
+            final BigDecimal totalDisbursed = loan.getDisbursedAmount().add(disbursedThroughCapitalizedIncome);
```

Comment:
Please dont mix disbursement amounts and capitalized income amounts! Keep them separate please!